### ISSUE TITLE
Fatal when `localStorage`-saved wallet does not rematerialize

### ISSUE THREAD

**Issue by steveluscher** [stevesarmiento/solana-starter#1]
Consider a wallet that doesn't conform to the Wallet Standard spec, like Brave Wallet (ie. it doesn't populate the `accounts` array on startup with the list of authorized accounts).

Repro:

1. Load site
2. Connect Brave Wallet
3. Refresh

The app remembers the last-connected wallet address, and tries to select it again on startup. Because Brave does not populate the `accounts` array, this account can not be found. Your app crashes with:

```
[Decoded] Wallet Standard error code #3834000
    - No underlying Wallet Standard wallet could be found for this handle. This can happen if the wallet associated with the handle has been unregistered.
```

**Comment by stevesarmiento** [comment]
@steveluscher fixed in [Commit 8366fc2
](https://github.com/stevesarmiento/solana-starter/commit/8366fc2390a9abf4589014cd0f6964303f1acb28)

**Comment by steveluscher** [comment]
Close! https://github.com/stevesarmiento/solana-starter/commit/8366fc2390a9abf4589014cd0f6964303f1acb28#commitcomment-149014986
